imports/ui/stylesheets/main.import.styl

```diff
@@ -8,6 +8,11 @@
   cursor hand
   color $secondary
 
+a.seasonal
+  color orange
```

Comment:
We should use the [primary variable] (https://github.com/ecohealthalliance/nia-monitor/blob/master/imports/ui/stylesheets/variables.import.styl#L3) to stay consistent with colors and such.